app/controllers/vacancies_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
       @vacancies_search.vacancies,
       searched: @vacancies_search.any?,
       total_count: @vacancies_search.vacancies.raw_answer['nbHits'],
-      coordinates: @vacancies_search.coordinates
+      coordinates: @vacancies_search.point_coordinates
```

Comment:
I think we can remove this line now

Reply:
Correction - we can't remove this (front end code this shows/hides radius input uses this data attribute)